src/gui/widgets/FxLine.cpp

```diff
@@ -25,6 +25,9 @@
 
 #include "FxLine.h"
 
+#include <cstdlib>
+#include <ctime>
+
```

Comment:
You can remove this as well as `rand()`.

Reply:
Uhh... why? I can remove ctime, but I use rand() for getting a random color, so I need that and cstdlib.

Reply:
I forgot the `rand()`.

Reply:
Okay, removed ctime.